cypress/integration/oauth2/authorize_code.js

```diff
@@ -3,91 +3,98 @@
 
 import { prng } from "../../helpers"
 
+const accessTokenStrategies = ["opaque", "jwt"]
```

Comment:
All OIDC and JWT suites now execute with an `opaque` and a `jwt` client. Since we start the suite twice, once with and without `--jwt`, we get the whole matrix tested between global and client access token strategy.

Reply:
🧠